VungleAdapter/src/main/java/com/chartboost/mediation/vungleadapter/VungleAdapter.kt

```diff
@@ -548,14 +549,24 @@ class VungleAdapter : PartnerAdapter {
         adOrientation?.let { adConfig.adOrientation = it }
 
         return suspendCancellableCoroutine { continuation ->
+            val continuationRef = WeakReference(continuation)
+
             fun resumeOnce(result: Result<PartnerAd>) {
-                if (continuation.isActive) {
-                    continuation.resume(result)
+                continuationRef.get()?.let {
```

Comment:
same here as meta. I'm not 100% sure this fixes it. you have access to continuation here

Reply:
Yeah, was no longer getting the leak with these changes. Also, I recently made the commit to move the listeners to their own class. For added measures.